src/components/Company/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './style.css';
+
+const Company = (props) => {
+
+    const isMilestone = Object.entries(props.milestone).length > 0;
+    console.log("isMilestone?", isMilestone);
+
+    const getRoles = (roles) => {
+        let roleArr = roles.map( (role, ind) =>
+            <ul key={ind}>
+                <li>{role}</li>
+            </ul>
+        )
+        return roleArr;
+    }
+
+    return (
+        <div className="company-info">
+            <div className="company-logo">
+                <img src={props.logo} />
+            </div>
+            <div className="details-section">
+                <div className="title">{props.title}</div>
+                <div>{props.subtitle}</div>
+                <div>{props.companyName}</div>
+                <div className="roles">Roles and Responsibilities</div>
+                {props.roles}
```

Comment:
why roles here as well as on line 34 ?

Reply:
34 is the milestone's role. means the milestone in KC (internship, then FTE - FE dev). So, the roles are different for both

Reply:
han toh specify that in the variable name na ?